Dockerfile

```diff
@@ -2,5 +2,18 @@ FROM python:3.7
 COPY ./requirements.txt /dockerBuild/requirements.txt
 WORKDIR /dockerBuild
 RUN pip install --no-cache-dir -r requirements.txt
+ENV DB_TYPE=postgresql
+ENV DB_USERNAME=postgres
+ENV DB_PASSWORD=postgres
+ENV DB_ENDPOINT=postgres:5432
+ENV DB_TEST_ENDPOINT=test_postgres:5432 
+ENV DB_NAME=mentorship_system
+ENV CI=true 
```

Comment:
Why this line is needed? Someone can run docker too for testing/reviewing instead of manually running and setting the code.

Reply:
Yeah removing only added it for testing